packages/terra-collapsible-menu-view/src/CollapsibleMenuViewToggle.jsx

```diff
@@ -99,7 +98,7 @@ class CollapsibleMenuViewToggle extends React.Component {
           labelText={text}
```

Comment:
The `type` prop isn't needed anymore here: https://github.com/cerner/terra-framework/pull/726/files#diff-973cc791b5a115b2e4130226902dded6R97

Reply:
Thanks..! correct we don't need this prop. Removed : f28770be24f893cbadc923f83b698207aba95d1b